cmd/convox/Makefile

```diff
@@ -14,8 +14,8 @@ build:
 package:
 	mkdir -p pkg/
 	export CGO_ENABLED=0
-	env GOOS=linux GOARCH=amd64 go build -ldflags "-X main.version=$(VERSION)" -o $(pkg_linux)
-	env GOOS=linux GOARCH=arm64 go build -ldflags "-X main.version=$(VERSION)" -o $(pkg_linux_arm64)
+	env GOOS=linux GOARCH=amd64 go build -tags=hidraw -ldflags "-X main.version=$(VERSION)" -o $(pkg_linux)
```

Comment:
What is `-tags=hidraw` doing?

Reply:
This tag changes the changes the USB HID to use the `hidraw` backend for `hidapi`, instead of `libusb`. It'll enable Linux users to use the Yubikey.
See [go-u2fhost Linux compatibility](https://github.com/convox/go-u2fhost#linux).